vowpalwabbit/core/src/reductions/cb/cb_algs.cc

```diff
@@ -135,96 +135,113 @@ void eval_finish_example(VW::workspace& all, cb& c, VW::example& ec)
   output_example(all, c, ec, ec.l.cb_eval.event);
   VW::finish_example(all, ec);
 }
-}  // namespace
 
-base_learner* VW::reductions::cb_algs_setup(VW::setup_base_i& stack_builder)
+struct options_cb_algs_v1
 {
-  options_i& options = *stack_builder.get_options();
-  VW::workspace& all = *stack_builder.get_all_pointer();
-  auto data = VW::make_unique<cb>(all.logger);
+  uint32_t num_actions;
   std::string type_string = "dr";
   bool eval = false;
   bool force_legacy = true;
+  size_t problem_multiplier = 2;  // default for DR
```

Comment:
Problem multiplier is a result of other options and not something actually configurable by a user. Based on that it seems like it shouldn't be in this struct?

Reply:
fair point, moved the problem_multiplier logic to setup